pyvis/network.py

```diff
@@ -526,7 +526,7 @@ def write_html(self, name, local=True, notebook=False):
 
             with open(f"{tempdir}/{name}", "w+") as out:
                 out.write(self.html)
-                webbrowser.open(f"{tempdir}/{name}")
+                # webbrowser.open(f"{tempdir}/{name}")
```

Comment:
Why not just remove it instead of keeping commented code?

Reply:
I saw that, for some reason, in the `show` method the call has been commented, therefore, I decided to comment it here as well. However, you're right and there is no need to leave this comment.